integration-test/Dockerfile.tester

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine@sha256:51b67269f354137895d43f3b3d810bfacd3945438e94dc5ac55fdac340352f48
```

Comment:
So moving forward we should always ref a sha for images and such?

Reply:
yeah its generally preferable - but probs we can let dependabot or similar take care of it